.github/workflows/ci.yml

```
@@ -0,0 +1,71 @@
+name: CI
+
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  test-hackatom:
+    name: Rust stable on ${{ matrix.build }}
```

Contract developers should always use Rust stable. With #504 this brings them even the same backend as used on-chain. Let's simplify this to

```suggestion
    name: ${{ matrix.build }}
```